### PR TITLE
Add raw_thread_member_remove event dispatch

### PR DESCRIPTION
**Pull request by 3nws**
## Summary

Adds a raw_thread_member_remove event dispatch and updates documentation reflecting the changes.

## Checklist

- [x] If code changes were made then they have been tested.
    - [x] I have updated the documentation to reflect the changes.
- [ ] This PR fixes an issue.
- [x] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [ ] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

File: discord/raw_models.py [hunk 1]
```diff
@@ -357,6 +359,32 @@ def __init__(self, data: ThreadDeleteEvent) -> None:
...
+    .. versionadded:: 2.0
+
+    Attributes
+    ----------
+    thread_id: :class:`int`
+        The ID of the thread that was updated.
+    guild_id: :class:`int`
+        The ID of the guild the thread is in.
+    member_count: :class:`int`
+        The number of members in the thread.
```
Comment (Rapptz) [inline discord/raw_models.py:374]: Should probably mention this is just for UI purposes and caps out at 50, similar to the `Thread.member_count` docstring.

File: discord/raw_models.py [hunk 2]
```diff
@@ -357,6 +359,32 @@ def __init__(self, data: ThreadDeleteEvent) -> None:
...
+    data: :class:`dict`
+        The raw data given by the :ddocs:`gateway <topics/gateway#thread-members-update>`
```
Comment (Rapptz) [inline discord/raw_models.py:376]:
```suggestion
        The raw data given by the :ddocs:`gateway <topics/gateway#thread-members-update>`.
```

File: discord/raw_models.py (hunk 1 above)
Comment (3nws) [reply discord/raw_models.py]: I wasn't sure if you meant to say `this is just for UI purposes` to me or for the docstring. English isn't my main language.

**Review by Rapptz — APPROVED** [review]
LGTM. Thanks.

**Pull request merged**